### DISCUSSION TITLE
Version 14.3.0 (release-tasmota) shows Core 2.7.7 ?

### DISCUSSION THREAD

**Discussion by askibit** [arendst/Tasmota#22489] [Q&A] [answered]
According to the changelog, version 14.3 should now use Core 2.7.8. After an OTA update of a Sonoff Basic, 2.7.7 is still displayed for me. Can someone please enlighten me? :-)

![image](https://github.com/user-attachments/assets/a3896c2f-62fc-4c99-bddc-c60520b403f2)

**Comment by Jason2866** [comment, marked as answer]
The only optical issue is fixed in latest development
